src/Raven.Client/Constants.cs

```diff
@@ -323,6 +323,17 @@ public class Folders
             }
         }
 
+        public class Smuggler
```

Comment:
we can make this class static, then remove constcutor.

Reply:
done